src/Engine/Driver.java

```diff
@@ -111,29 +106,33 @@ public static void main(String args[])
   @Override // From KeyListener
   public void keyPressed(KeyEvent arg0)
   {
+    // Store off the old controller/view so we can return to them.
+    IView oldView = gameView.view;
```

Comment:
The scope of `oldView` could be reduced. Also, I'm not sure the comment here is accurate to how `oldView` is being used.